README.md

```diff
@@ -24,7 +24,7 @@ website](https://docs.astarte-platform.org/device-sdks/esp32/latest/api).
 
 The SDK is tested against these versions of `esp-idf`:
 - `v4.4`
-- `v5.0`
+- `v5.x`
```

Comment:
Is it right to declare that will support all the version 5.x?
On the [esprissif-docs](https://docs.espressif.com/projects/esp-idf/en/latest/esp32/migration-guides/release-5.x/5.2/index.html) you can see already some migration guide to 5.2 or 5.3, that makes me think we will have a breaking changes, so it is better to keep `5.0, 5.1`.